src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsRequest.js

```diff
@@ -141,6 +141,7 @@ class IOUParticipantsRequest extends Component {
                 headerMessage={headerMessage}
                 placeholderText={this.props.translate('optionsSelector.nameEmailOrPhoneNumber')}
                 boldStyle
+                safeAreaPaddingBottomStyle={this.props.safeAreaPaddingBottomStyle}
```

Comment:
Seems to be missing in propTypes too

Reply:
Good catch thanks for jumping in here @aldo-expensify!

Reply:
done